### ISSUE TITLE
Nuxt 3 Support

### ISSUE THREAD

**Issue by HelloAlexPan** [nuxt-community/eslint-module#69]
Opening an issue to discuss whether there's plans to support Nuxt 3 and how the community can help.

**Comment by ThomasBerne** [comment]
The same for nuxt bridge. It doesn't seems there is support about it.

**Comment by ricardogobbosouza** [comment]
Starting support nuxt3/bridge https://github.com/nuxt-community/eslint-module/pull/70

**Comment by jonorozcoc** [comment]
Any update on this?

**Comment by mlbiche** [comment]
@jonorozcoc 

> Any update on this?

Haven't tried but #68 mentions that it works

**Comment by jonorozcoc** [comment]
Thanks @mlbiche

I'll check it

**Comment by mlbiche** [comment]
At the end, I am using `eslint-plugin-nuxt`, `eslint-plugin-vue` and `@vue/eslint-config-standard`. I have removed all Nuxt modules related to ESLint and all babel-related packages. And I have updated my ESLint config from something like:

```js
module.exports = {
  root: true,
  env: {
    'browser': true,
    'node': true
  },
  parserOptions: {
    parser: '@babel/eslint-parser',
    requireConfigFile: false
  },
  extends: ['@nuxtjs', 'plugin:nuxt/recommended']
}
```

... to something like:

```
module.exports = {
  root: true,
  env: {
    'browser': true,
    'node': true
  },
  parserOptions: {
    ecmaVersion: 'latest',
    sourceType: 'module'
  },
  extends: [
    'plugin:nuxt/recommended',
    'plugin:vue/vue3-recommended'
  ],
  overrides: [
    {
      files: ['layouts/*.vue', 'pages/**/*.vue'],
      rules: { 'vue/multi-word-component-names': 'off' }
    }
  ]
}
```

**Comment by jonorozcoc** [comment]
@mlbiche is the plugin nuxt/recommended up-to-date with nuxt3?

**Comment by mlbiche** [comment]
@jonorozcoc At least it does not raise false positives with Nuxt 3. But it seems that it provides fewer rules than this module.

**Comment by ricardogobbosouza** [comment]
Nuxt3 Support in progress https://github.com/nuxt-community/eslint-module/pull/70

**Comment by ricardogobbosouza** [comment]
Released [v4.0.0](https://github.com/nuxt-community/eslint-module/releases/tag/v4.0.0)
